### PR TITLE
Temp New Officer List

### PR DESCRIPTION
**Pull request by TylerAdamMartinez**
Feel free to review it and check it, if you want anything to be changed let me know.

### CODE REVIEW

**Comment by ntindle** [comment]
LGTM

**Comment by ntindle** [comment]
@TylerAdamMartinez  for an IDE, I use VS Code which has autoformatters for php. Its pretty simple to set up. Open the root level directory and it should have you install a bunch of extensions for php. Then run `Alt + Shift + F` on each of the files you pushed. It'll auto format it. 

I can add a style reviewer to the repo if you'd like @sebastian-king. lmk

File: our-team.php [hunk 1]
```diff
@@ -3,180 +3,185 @@
...
+        </div>
+      </div>
+      <div class="range">
+        <div class="cell-lg-6 cell-sm-6 cell-md-12">
+          <div class="range range-sm-middle">
+            <div class="cell-md-6"><img src="images/bio-pics/juan-ruiz.jpg" alt="" width="360" height="404" class="img-responsive" />
+            </div>
+            <div class="cell-md-6">
+              <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
+                <h6 class="h6-with-small"><a href="#"> Juan Ruiz</a><span class="small text-silver-chalice">President</span></h6>
```
Comment (ntindle) [inline our-team.php]: Add a little comment for each person's section saying who it is so I don't have to dig in as far.

File: our-team.php [hunk 2]
```diff
@@ -3,180 +3,185 @@
...
+                  <p>I am the Webmaster for UNT Robotics. I'm in charge of the website that you see in front of you. I also integrate our technology stack with other organisations and maintain the online equipment checkout system.</p>
+                  <p>I'm almost finished studying Computer Science and Electrical Engineering, so robotics is the natural choice for all of my talents. I love to be hands on with technology, and take apart and break anything that I can. For me there's nothing quite like finally seeing a project successfully working out in the real-world.
+                  </p>
+                  <ul class="list-inline-lg">
+                    <li><a href="https://www.linkedin.com/in/sebastian-king" class="icon icon-sm text-primary fa-linkedin"></a></li>
+                    <li><a href="https://www.twitter.com/@thekingseb" class="icon icon-sm text-primary fa-twitter"></a></li>
+                    <li><a href="https://www.github.com/sebastian-king" class="icon icon-sm text-primary fa-github"></a></li>
+                    <li><a href="mailto:webmaster@untrobotics.com" class="icon icon-sm text-primary fa-envelope"></a></li>
```
Comment (ntindle) [inline our-team.php]: Do we need this? Should we have this for everyone?

**Comment by ntindle** [comment]
also run a spell checker on it just in case

**Comment by TylerAdamMartinez** [comment]
How? 
Btw, I did use VScode did know of the command still getting used to the IDE

**Comment by TylerAdamMartinez** [comment]
Okay, all the officer pics are in that folder. Sorry I thought they were all in the folder

**Comment by TylerAdamMartinez** [comment]
That command is not working for me. Do I have to change the config.json or is that part of an extension that you downloaded already

**Comment by ntindle** [comment]
You need `PHP Intelephense`. Sorry.

File: our-team.php [hunk 3]
```diff
@@ -3,180 +3,185 @@
...
+  </section>
+  <section class="section-75 section-md-100 section-lg-150">
+    <div class="shell range-offset-1">
+      <div class="range">
+        <div class="cell-lg-6">
+          <h1>Our Team</h1>
+          <h6>These people are the reason for our success and expertise.</h6>
+        </div>
+      </div>
+      <div class="range">
```
Comment (ntindle) [inline our-team.php]:
```suggestion
<! -- This is a Juan -->
            <div class="range">

```

File: our-team.php (hunk 2 above)
Comment (sebastian-king) [reply our-team.php]: Yes we absolutely should have those for everyone.

Comment (ntindle) [reply our-team.php]: To be clear, Emails

Comment (sebastian-king) [reply our-team.php]: Yes, everyone should have an @untrobotics.com email under their name.

File: our-team.php (hunk 3 above)
Comment (sebastian-king) [reply our-team.php]: Yes please, those would be great.

Comment (TylerAdamMartinez) [reply our-team.php]: IT was supposed to say comments

**Comment by sebastian-king** [comment]
@ntindle what specifically needs to be updated by me?

**Comment by ntindle** [comment]
You’re a German now

**Comment by TylerAdamMartinez** [comment]
PHP is now in my path; therefore, it should work
@sebastian-king @ntindle Can one of you use your auto indents and finishes this

**Comment by TylerAdamMartinez** [comment]
After the indents what is left to resolve before y'all will commit

**Comment by ntindle** [comment]
LGTM

________________________________

**Comment by ntindle** [comment]
Do you have a mail server setup

________________________________

**Comment by sebastian-king** [comment]
We do, I'll have @untrobotics.com up by the time this PR is merged.

**Comment by TylerAdamMartinez** [comment]
Does everyone have robotics email or are you going to make them after I update the page

**Comment by TylerAdamMartinez** [comment]
@sebastian-king

**Comment by sebastian-king** [comment]
We will have them before the page is updated

**Pull request merged**